kafka-admin/src/main/java/io/strimzi/admin/kafka/admin/handlers/TopicHandler.java

```diff
@@ -0,0 +1,23 @@
+package io.strimzi.admin.kafka.admin.handlers;
+
+import graphql.schema.DataFetchingEnvironment;
+import io.strimzi.admin.kafka.admin.model.Topic;
+import io.vertx.core.Promise;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TopicHandler {
+    private static final Map<String, Topic> sampleTopics = new HashMap<>();
```

Comment:
constants should be all upper case.

Reply:
Changed it to be uppercase but again, it is a temporary example.